store/vars.ts

```diff
@@ -0,0 +1,31 @@
+import { type Step, type RoutineTimeVariationType } from "./types"
```

Comment:
I put in this file the starting data of the webapp. In this way they can be imported without using pinia stores data.